Makefile

```diff
@@ -1,26 +1,66 @@
+.PHONY: help venv conda docker docstyle format style black test lint check
```

Comment:
If we're going to use `.PHONY` here I think we should be consistent and use it everywhere

Reply:
What do you mean by everywhere? In other Makefiles?

Reply:
On every target. For example line 9, line 13, line 38 etc.

Eric pointed out in his original PR that it wasn't needed, but I'm somewhat a stickler for consistency

Reply:
Or maybe, I just realized, I don't understand makefiles well enough and by specifying it on line one, that's the equivalent of specifying it for everything.........

Reply:
Ah I see! In Makefiles `.PHONY` is a special reserved variable that indicates what recipes are named (so that Make doesn't confuse them for file names), so specifying them up front is all you need to do (it doesn't even need to be on line 1).